ReadMe.md

```diff
@@ -1,11 +1,39 @@
+# Kotlin Programming Language
+
 [![official project](https://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![TeamCity (simple build status)](https://img.shields.io/teamcity/http/teamcity.jetbrains.com/s/Kotlin_KotlinPublic_Compiler.svg)](https://teamcity.jetbrains.com/buildConfiguration/Kotlin_KotlinPublic_Compiler?branch=%3Cdefault%3E&buildTypeTab=overview&mode=builds)
 [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin/kotlin-maven-plugin.svg)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.jetbrains.kotlin%22)
 [![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](https://www.apache.org/licenses/LICENSE-2.0)
 [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jetbrains.com/scans?search.rootProjectNames=Kotlin)
 
-# Kotlin Programming Language
+<img width="250" height="250" alt="Microsoft(1)" src="https://github.com/user-attachments/assets/7554978f-82f3-474c-a422-cd401c49068f" />
```

Comment:
> <img width="50" height="50" src="https://github.com/user-attachments/assets/7554978f-82f3-474c-a422-cd401c49068f">

The Kotlin logo is incorrect. As far as I remember, this logo was only used in the early days of Kotlin, and the current logo can be found [here](https://kotlinlang.org/docs/kotlin-brand-assets.html#kotlin-logo).

Also, what does the “Microsoft” in the alt text mean?